### ISSUE TITLE
Wrong object reference in README.md

### ISSUE THREAD

**Issue by Sephtenen** [alexa-js/alexa-app#360]
I'm currently using alexa-app on Lambda.

[The doc says](https://github.com/alexa-js/alexa-app#dialog-object) there is a `dialogState` property on the request object :

```
// return the intent's dialogState
String request.dialogState
```

I found this property to be `undefined`. However, it is defined (and correct) in the `dialog` object.

So the above example should be :

```
// return the intent's dialogState
String dialog.dialogState
```

**Comment by dblock** [comment]
Maybe PR a fix? Is it just a typo? Do we have code doing that? Thanks.

**Comment by Sephtenen** [comment]
Yes, it's basically a typo imho, as other similar objects ([slot](https://github.com/alexa-js/alexa-app#slot) and [slotResolution](https://github.com/alexa-js/alexa-app#slotresolution)) description follows a pattern like :
```
var <name> = <get>
<type> <propName1> = <name>.<property1>
<type> <propName2> = <name>.<property2>
<type> <propName3> = <name>.<property3>
...
```
